src/js/puzzles.js

```diff
@@ -12,7 +12,28 @@ var puzzlesArray = [
         word: 'a sheep',
         hint: 'tired',
         answer: 'asleep'
-    }
+    },
+    {
+        id: 3,
+        sentence: 'I will go grocery stopping tomorrow.',
+        word: 'stopping',
+        hint: 'chore',
+        answer: 'shopping',
+    },
+    {
+        id: 4,
+        sentence: 'I like to play with my pat.',
+        word: 'pat',
```

Comment:
`Pat` and `pet` don't rhyme, so this answer won't quite fit into our theme. You could use `met` or `set` or something similar.